zipkin-storage/influxdb/src/main/java/zipkin/storage/influxdb/InfluxDBStorage.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2016-2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.storage.influxdb;
+
+import org.influxdb.InfluxDB;
+import zipkin2.Call;
+import zipkin2.DependencyLink;
+import zipkin2.Span;
+import zipkin2.storage.QueryRequest;
+import zipkin2.storage.SpanConsumer;
+import zipkin2.storage.SpanStore;
+import zipkin2.storage.StorageComponent;
+
+import java.util.List;
+
+public class InfluxDBStorage extends StorageComponent implements SpanStore, SpanConsumer {
+
+  public static Builder newBuilder() {
+    return new Builder();
+  }
+
+  public static final class Builder extends StorageComponent.Builder {
+    boolean strictTraceId = true;
+    String address;
```

Comment:
set defaults here instead of in in the properties type. for the properties type https://github.com/openzipkin/zipkin/blob/master/zipkin-autoconfigure/collector-rabbitmq/src/main/java/zipkin/autoconfigure/collector/rabbitmq/ZipkinRabbitMQCollectorProperties.java is a better example as it ensures you only have to change properties once (here)